test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -696,7 +696,7 @@ func TestProjectWithPrivateEndpoint_Azure(t *testing.T) {
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
 		require.NoError(t, err)
-		s.t.Cleanup(func() {
+		t.Cleanup(func() {
```

Comment:
This makes sense to me, the `t` here is not guaranteed to be the same as from the line 673